MobileCenter/MobileCenter/Internals/Utils/MSKeychainUtil.h

```diff
@@ -31,15 +52,33 @@ NS_ASSUME_NONNULL_BEGIN
  * Get a string from Keychain with the given key and service name.
  *
  * @param key A unique key for the data.
+ *
+ * @return A string data if exists.
+ */
++ (NSString *)stringForKey:(NSString *)key;
+
+/**
+ * Get a string from Keychain with the given key and service name.
+ *
+ * @param key A unique key for the data.
  * @param service A service name for the key.
+ *
  * @return A string data if exists.
  */
 + (NSString *)stringForKey:(NSString *)key service:(NSString *)service;
 
 /**
  * Clear all keys and strings associated with the given service name.
  *
+ * @return YES if cleared successfully, NO otherwise.
+ */
++ (BOOL)clearForService;
```

Comment:
Changing this to use mainBundle. We don't need to have service name as a parameter.
This needs to be done in `develop`.